app/i18n/en/admin.php

```diff
@@ -138,7 +138,7 @@
 		'number_entries' => '%d articles',
 		'percent_of_total' => '%% of total',
 		'repartition' => 'Articles repartition',
-		'status_favorites' => 'Favourites',
+		'status_favorites' => 'Favorites',
```

Comment:
Both are correct. One is for US, the other one is for UK. I have no preference even though I am more used to "Favourites"